client/src/components/ExpandingFormset/index.js

```diff
@@ -28,16 +28,24 @@ export class ExpandingFormset {
     });
   }
 
-  addForm() {
+  addForm(opts = {}) {
+    /*
+    Supported opts:
+    runCallbacks (default: true) - if false, the onAdd and onInit callbacks will not be run
+    */
+    const formIndex = this.formCount;
```

Comment:
```suggestion
  /**
   * @param {string} opts - Options
   * @param {boolean) [runCallbacks=true] - if false, the onAdd and onInit callbacks will not be run
   */
  addForm(opts = {}) {
    const formIndex = this.formCount;
```

Not 100% on syntax but should be close enough once you run the linter, using JSDOC here will help for editor auto-completion instead of non-standard commenting syntax.

https://jsdoc.app/